src/System.ComponentModel.TypeConverter/src/System/misc.cs

```diff
@@ -0,0 +1,197 @@
+            //     // Someone is trying to instantiate a public type in *our* assembly, but does not
+            //     // have full reflection permission. We shouldn't pass BindingFlags.NonPublic in this case.
+            //     // The reason we don't directly demand the permission here is because we don't know whether
+            //     // a public or non-public .ctor will be invoked. We want to allow the public .ctor case to
+            //     // succeed.
+            //     allowNonPublic = false;
+            // }
+
+            // if (allowNonPublic) {
+            //     flags |= BindingFlags.NonPublic;
+            // }
+
+            return Activator.CreateInstance(type, flags, null, args, null);
+        }
+    }
+
+    internal static class HResults
+    {
+        internal const int License = unchecked((int)0x80131901);
+    }
+}
+
+namespace System.Resources
+{
+    public partial interface IResourceReader : System.Collections.IEnumerable, System.IDisposable
+    {
+        void Close();
+        new System.Collections.IDictionaryEnumerator GetEnumerator();
+    }
+    public partial interface IResourceWriter : System.IDisposable
+    {
+        void AddResource(string name, byte[] value);
+        void AddResource(string name, object value);
+        void AddResource(string name, string value);
+        void Close();
+        void Generate();
+    }
+
+    public partial class ResourceSet : System.Collections.IEnumerable, System.IDisposable
+    {
+        protected System.Resources.IResourceReader Reader;
+        protected System.Collections.Hashtable Table;
+        protected ResourceSet() { }
+        public ResourceSet(System.IO.Stream stream) { }
+        public ResourceSet(System.Resources.IResourceReader reader) { }
+        public ResourceSet(string fileName) { }
+        public virtual void Close() { }
+        public void Dispose() { }
+        protected virtual void Dispose(bool disposing) { }
+        public virtual System.Type GetDefaultReader() { throw null; }
+        public virtual System.Type GetDefaultWriter() { throw null; }
+        public virtual System.Collections.IDictionaryEnumerator GetEnumerator() { throw null; }
+        public virtual object GetObject(string name) { throw null; }
+        public virtual object GetObject(string name, bool ignoreCase) { throw null; }
+        public virtual string GetString(string name) { throw null; }
+        public virtual string GetString(string name, bool ignoreCase) { throw null; }
+        protected virtual void ReadResources() { }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+}
+
+internal static class AssemblyRef
+{
+    internal const string SystemDrawing = "System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
+    internal const string SystemDesign = "System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
+}
```

Comment:
Removed them from here. they are used inside DesignerAttribute and EditorAttributes. I moved the string literals directly there. We might want to remove them completely.